### PR TITLE
Added build script for setprototypeof.

### PR DESCRIPTION
**Pull request by bhagat-singh001**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: s/setprototypeof/setprototypeof_v1.1.1_rhel_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
+#!/bin/bash -e
+
+# -----------------------------------------------------------------------------
+#
+# Package           : setprototypeof
+# Version           : v1.1.1
```
Comment (seth-priya) [inline s/setprototypeof/setprototypeof_v1.1.1_rhel_8.5.sh:6]: is the script specific to this version or will work for top-of-tree as well? If its the latter, can we remove version from script name to make it generic?

Comment (bhagat-singh001) [reply s/setprototypeof/setprototypeof_v1.1.1_rhel_8.5.sh]: renamed the file as suggested.

**Comment by bhagat-singh001** [comment]
Reopen PR as there are some error in Travis job.

**Comment by bhagat-singh001** [comment]
The job exceeded the maximum time limit for jobs, and has been terminated.

**Comment by bhagat-singh001** [comment]
The job exceeded the maximum time limit for jobs, and has been terminated.

**Comment by bhagat-singh001** [comment]
The job exceeded the maximum time limit for jobs, and has been terminated.

**Comment by bhagat-singh001** [comment]
The job exceeded the maximum time limit for jobs, and has been terminated.

**Comment by bhagat-singh001** [comment]
The job exceeded the maximum time limit for jobs, and has been terminated.

**Comment by bhagat-singh001** [comment]
The job exceeded the maximum time limit for jobs, and has been terminated.

**Review by seth-priya — APPROVED** [review]

**Pull request merged**